Enable `TestAltinityConnectorSmokeTest` in ClickHouse connector

ebyhr opened the pull request:
## Description

Enable `TestAltinityConnectorSmokeTest` in ClickHouse connector

## Related issues, pull requests, and links

n/a

## Documentation

(x) No documentation is needed.

## Release notes

(x) No release notes entries required.

ebyhr commented:
50 stress tests succeeded https://github.com/trinodb/trino/runs/5242608573

hashhar reviewed (approved):
This works because we updated version of Altinity we test against and we can now use newer driver with it which doesn't cause flakiness?

If my understanding is correct can you please add this to the commit message. It'll not be  obvious why or how this got fixed in future.

ebyhr commented:
> This works because we updated version of Altinity we test against and we can now use newer driver with it which doesn't cause flakiness?

Right, updated the commit message.
